src/__test__/__mocks__/MockHTML.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable import/prefer-default-export */
```

Comment:
- [ ] The linters are there to enforce code quality and should not be disabled. Kindly enable the linters and fix the issues raised. In this case, check that the props are validated.

Reply:
Thanks for it, I just forgot to enable them.